spring-context-support/src/main/java/org/springframework/cache/jcache/JCacheCache.java

```diff
@@ -28,7 +28,7 @@
 
 /**
  * {@link org.springframework.cache.Cache} implementation on top of a
- * {@link javax.cache.Cache} instance.
```

Comment:
Without the fully qualified package name, we'd want to have "JCache" there in addition.

So we might as well just leave it as is for clarity, as was originally intended.

Thus, please either undo this change or prefix that with "JCache" as is done elsewhere in the Javadoc for this class.

Reply:
@sbrannen It looks the same concern as @snicoll 's https://github.com/spring-projects/spring-framework/pull/1938#discussion_r212008591. I answered there and let me know what you think.